tests/Gedmo/Translator/TranslatableTest.php

```diff
@@ -102,7 +102,7 @@ public function testShouldTranslateRelation(): void
         $person->translate('ru')->setDescription('multilingual description');
 
         $parent = new Person();
-        $parent->setName('Jen');
+        $parent->setName('Jen parent');
```

Comment:
In this case when calling https://github.com/doctrine-extensions/DoctrineExtensions/blob/4fd6a62deb3f4406435e5a869f3b33e7560f5c11/tests/Gedmo/Translator/TranslatableTest.php#L117 it fetches the first one persisted, here we're interested in fetching the child.